stock_cycle_count/models/stock_cycle_count_rule.py

```diff
@@ -231,7 +228,7 @@ def _compute_rule_turnover(self, locs):
                 self.env["stock.inventory"]
                 .search(
                     [
-                        ("location_id", "=", loc.id),
+                        ("location_ids", "child_of", [loc.id]),
```

Comment:
same here. This should not include childs, it should be "in" instead of "child_of"